src/lib/utils.ts

```diff
@@ -565,7 +576,10 @@ export function hashData(
   data: string | Uint8Array | Record<string, any>,
   nameOrSig: SUPPORTED_VERIFICATION_METHODS | string,
 ): string {
-  return getVerificationMethod(nameOrSig)?.method(data) || leftPad(0, 64);
+  return (
+    getVerificationMethod(nameOrSig)?.method(data) ||
+    pad(toHex(0), { size: 32, dir: 'left' })
```

Comment:
Suggest using this function

```suggestion
    numberToHex(0, { size: 32 })
```

<img width="748" alt="image" src="https://github.com/user-attachments/assets/888be73a-410b-4ac7-94c8-221573618d60" />